### DISCUSSION TITLE
Release  22000.318.37.2

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#357] [General]
After clicking the setup, it uninstalled EP. Then I clicked second time and it installed. But

* Reappeared the notification center icon
* Reappeared the show desktop icon
* Disappeared the quick launch.

Possibly, this all is due to switching to Win11 taskbar.

The release otherwise seems a great progress!

**Comment by Anixx** [comment]
I also suggest to make more descriptive text labels for options. For instance "Register as shell extension" can be confusing, so I suggest adding a clarification ("needed for effect on file dialogs") or something like this.

The point "Enable SysListView32" also could be more descriptive (in Folder Options X it is described as "Enable icon reordering").

**Reply by valinet** [reply]
![image](https://user-images.githubusercontent.com/6503598/141713934-87b80fbd-4ade-4894-90c1-fb90d6b40988.png)

It's literally below it .......

**Reply by Anixx** [reply]
Ah, I did not notice this, apologies!

**Comment by valinet** [comment]
> 
> 
> After clicking the setup, it uninstalled EP. Then I clicked second time and it installed. But
> 
>     * Reappeared the notification center icon
> 
>     * Reappeared the show desktop icon
> 
>     * Disappeared the quick launch.
> 
> 
> Possibly, this all is due to switching to Win11 taskbar.
> 
> The release otherwise seems a great progress!

Yeah, it does that when the Windows 11 taskbar gets enabled. Basically, the W11 taskbar disables any toolbars. The rest is because I added some code to keep them available when the W11 taskbar appears, but one time it may make them disappear. It's one time though.

Manual update is done by running `ep_setup.exe /update_silent`, so you don't necessarily need to uninstall first.
